### ISSUE TITLE
 ERROR: aten::norm is not Supported!

### ISSUE THREAD

**Issue by jia0511** [microsoft/nni#5018]
When I use the NNI L1 method to prune the facenet model and perform speedup, I encounter an error indicating that aten::norm is not supported. How to solve this problem?My pytorch network structure is defined as follows：

`class Facenet(nn.Module):
    def __init__(self, backbone="mobilenet", dropout_keep_prob=0.5, embedding_size=128, num_classes=None, mode="train", pretrained=False):
        super(Facenet, self).__init__()
        if backbone == "mobilenet":
            self.backbone = mobilenet(pretrained)
            flat_shape = 1024
        elif backbone == "inception_resnetv1":
            self.backbone = inception_resnet(pretrained)
            flat_shape = 1792
        else:
            raise ValueError('Unsupported backbone - `{}`, Use mobilenet, inception_resnetv1.'.format(backbone))
        self.avg        = nn.AdaptiveAvgPool2d((1,1))
        self.Dropout    = nn.Dropout(1 - dropout_keep_prob)
        self.Bottleneck = nn.Linear(flat_shape, embedding_size,bias=False)
        self.last_bn    = nn.BatchNorm1d(embedding_size, eps=0.001, momentum=0.1, affine=True)
        if mode == "train":
            self.classifier = nn.Linear(embedding_size, num_classes)

    def forward(self, x, mode = "predict"):
        if mode == 'predict':
            x = self.backbone(x)
            x = self.avg(x)
            x = x.view(x.size(0), -1)
            x = self.Dropout(x)
            x = self.Bottleneck(x)
            x = self.last_bn(x)
            x = F.normalize(x, p=2, dim=1)
            return x
        x = self.backbone(x)
        x = self.avg(x)
        x = x.view(x.size(0), -1)
        x = self.Dropout(x)
        x = self.Bottleneck(x)
        before_normalize = self.last_bn(x)`

**Comment by jia0511** [comment]
![2022-07-25 17-08-51](https://user-images.githubusercontent.com/24822418/180741123-ef0d3002-ff62-47b7-92ad-a67c0550288f.png)

**Comment by jia0511** [comment]
![2022-07-25 17-10-50](https://user-images.githubusercontent.com/24822418/180741514-4d81fcae-b797-40e5-a40f-faf8bd578633.png)

**Comment by jia0511** [comment]
@[Louis-J](https://github.com/Louis-J) 
I think you can help me as soon as possible, thank you!

**Comment by Louis-J** [comment]
have you tried replacing the `{envs_path}\Lib\site-packages\nni\compression\pytorch\speedup\jit_translate.py` to the [new_dev_version](https://github.com/Louis-J/nni/blob/dev-speedup-auto-op/nni/compression/pytorch/speedup/jit_translate.py)?

**Comment by jia0511** [comment]
I ran into this problem using NNI version 2.8.

**Comment by jia0511** [comment]
@Louis-J 
>> have you tried replacing the {envs_path}\Lib\site-packages\nni\compression\pytorch\speedup\jit_translate.py to the [new_dev_version](https://github.com/Louis-J/nni/blob/dev-speedup-auto-op/nni/compression/pytorch/speedup/jit_translate.py)?

I tried replacing the jit_translate.py ,but met new problem:
![2022-07-27 13-53-31](https://user-images.githubusercontent.com/24822418/181171972-4c1a55da-a0f9-4378-b03e-6a3e77eb4a84.png)

**Comment by jia0511** [comment]
> ![2022-07-25 17-10-50](https://user-images.githubusercontent.com/24822418/180741514-4d81fcae-b797-40e5-a40f-faf8bd578633.png)

I would like to know if there is a good solution to this problem？

**Comment by jia0511** [comment]
Hi @Louis-J

I would like to know whether the new version will solve this problem, and whether the reason for the speedup error is related to the model structure or NNI. When I do not do the speedup model, the model can be executed normally.

**Comment by jia0511** [comment]
Hi @Louis-J

Is there an update on this issue, is the 2.9 version fixed to support norm?

**Comment by Louis-J** [comment]
Sorry for not replying these days. Now in 2.9 version we submitted the feature to automatically using the ops in `aten::` namespace. the aten::norm is supported now.

But there is a bug still that we cannot use the 'expand_as' function correctly. And all the returned view may cause the problem. We will fix this bug in 2.9.1.

**Comment by Louis-J** [comment]
the `expand_as` is solved in #5141 included in version 2.10. you can try with the latest version 2.10 now.
